docs/docusaurus/netlify/functions/createJiraTicketInDocsBoard.js

```diff
@@ -0,0 +1,50 @@
+const TITLE_MAX_CHARACTERS = 120;
+const DOCUMENTATION_BOARD_ID = 10019;
+const STORY_ISSUETYPE_ID = 10011;
+const CREATE_JIRA_ISSUE_ENDPOINT_URL = "https://greatexpectations.atlassian.net/rest/api/2/issue"
+
+const truncateDescription = (description) => {
+    return description.length > TITLE_MAX_CHARACTERS ? description.substring(0, TITLE_MAX_CHARACTERS) + "..." : description;
+}
+
+const formatOptionalValue = (value) => value || '-';
+
+const fullDescription = (description, name, email, selectedValue) => {
+    const formattedSelectedValue = selectedValue.replaceAll("-"," ");
+    return `Name: ${formatOptionalValue(name)}\nEmail: ${formatOptionalValue(email)}\nSelected feedback type: ${formattedSelectedValue}\n\n${description}`;
+}
+
+exports.handler = async (req, context) => {
+    const { description, name, email, selectedValue } = JSON.parse(req.body);
+    try {
+        const response = await fetch(CREATE_JIRA_ISSUE_ENDPOINT_URL, {
+            method: "POST",
+            headers: {
+                'Content-Type': 'application/json',
+                'Authorization': 'Basic ' + Buffer.from(process.env.JIRA_API_USER + ":" + process.env.JIRA_API_TOKEN).toString('base64')
```

Comment:
Let's wait until we have this service user situation buttoned up before we merge this.